UNIX/build.sh

```diff
@@ -22,6 +22,12 @@ if [[ ("$LNT" != "" || "$BUILD_PACKAGE" == "Yes") &&
   CMAKE_ADDITIONAL_OPTIONS="-DLLVM_INSTALL_TOOLCHAIN_ONLY=ON -DLLVM_ENABLE_ASSERTIONS=ON"
 fi
 
+if [[ "$CLANGD" == "Yes" ]]; then
+  ENABLEPROJECTS="clang;clang-tools-extra"
```

Comment:
Does building `clang-tools-extra` require any additional repo to be cloned?

Reply:
No, I checked that. No additional cloning is required.